src/compas/scene/sceneobject.py

```diff
@@ -176,21 +169,17 @@ def transformation(self, transformation):
     @property
     def worldtransformation(self):
         # type: () -> compas.geometry.Transformation
-        frame_stack = [self.frame] if self.frame else []
-        parent = self.parent
+        transformations = [self.transformation] if self.transformation else []
```

Comment:
although probably nobody was using this, there is a slight change in behaviour that should be documented:

there can no longer be a independent transformation wrt to the local frame...

Reply:
sure, I added a note also mentioning now in changelog